packages/flutter/lib/src/widgets/form.dart

```diff
@@ -579,8 +614,12 @@ class FormFieldState<T> extends State<FormField<T>> with RestorationMixin {
     Form.maybeOf(context)?._fieldDidChange();
   }
 
-  /// Calls [FormField.validator] to set the [errorText]. Returns true if there
-  /// were no errors.
+  /// Calls [FormField.validator] to set the [errorText] only if [FormField.forceErrorText] is null.
+  /// Returns true if there were no errors.
+  ///
+  /// Note: When using the [FormField.forceErrorText] property directly to set the
```

Comment:
Another "Note" https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-empty-prose

Reply:
removed this too.